shell/common/rasterizer.cc

```diff
@@ -259,6 +259,10 @@ RasterStatus Rasterizer::DrawToSurface(flutter::LayerTree& layer_tree) {
     }
     frame->Submit();
     if (external_view_embedder != nullptr) {
+      // Frame submission will only flush the surface canvas. But we are
+      // rendering into a canvas managed by the external view embedder. This
+      // must be flushed manually.
+      root_surface_canvas->flush();
```

Comment:
As discussed - move this to EmbedderExternalViewEmbedder::SubmitFrame to avoid double flush on iOS

Reply:
Done.